packages/website/src/components/CurriculumSection/TechSliding.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { Box } from '@coderscamp/ui/components/Box';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { keyframes } from '@coderscamp/ui/components/Keyframes';
+
+import { techNames } from './TechIcons';
+
+interface TechSlidingProps {
+  direction?: 'normal' | 'reverse';
+}
+
+export const TechSliding = ({ direction = 'normal' }: TechSlidingProps) => {
+  const sliding = keyframes`
+   100% {
+    transform: translateX(-66.6666%);
+  }
+`;
+  const animation = `${sliding} infinite 60s linear ${direction}`;
+
+  return (
+    <Box w="100%" bg="gray.100" h="140px" sx={{ position: 'relative', overflow: 'hidden' }}>
+      <Flex
+        w="300%"
+        h="100%"
+        align="center"
+        justify="space-between"
+        animation={animation}
+        sx={{ position: 'absolute', top: 0, left: 0, transform: 'translateX(0)' }}
+      >
+        {techNames.map((tech) => (
+          <Box fontSize={{ base: '52px', lg: '78px' }} px={{ base: '32px', lg: 0 }} key={tech.name}>
```

Comment:
🤔 It looks like those icons aren't centered vertically (they are closer to the bottom)
<img width="818" alt="CleanShot 2021-08-17 at 22 45 16@2x" src="https://user-images.githubusercontent.com/22938071/129798157-88626209-21b6-495b-a24f-cb88c21bfe17.png">

Reply:
💭 Great idea to change the `fontSize` on smaller devices. I think you can also make the entire bar size a little smaller